wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php

```diff
@@ -106,7 +125,7 @@ function SSAPDFadminBuildPDF ()
 		switch_to_blog( $blogID );
 	}
 	
-	if ( current_user_can( 'create_and_download_pdf' ) ) //Only let them download the file if they are allowed to
+	if ( current_user_can( 'create_and_download_pdf' ) ) //Only let them download the file if they are admin
```

Comment:
I don't even remember doing those changes. Maybe this is due to a software update. I've recently downloaded the plugin and modified it to satisfy our needs. But I guess everyone can do so. I found those lines:
<pre>
add_action( 'wp_ajax_frontEndDownloadPDF', 'frontEndDownloadPDF' );			// fires if logged-in user
add_action( 'wp_ajax_nopriv_frontEndDownloadPDF', 'frontEndDownloadPDF' );	// fires if non-logged-in user
</pre>